metriq_gym/circuits.py

```diff
@@ -34,6 +35,25 @@ def qiskit_to_tk(circ: QuantumCircuit) -> Circuit:
     return _qiskit_to_tk(circ)
 
 
+def qiskit_to_qrack(circ: QuantumCircuit) -> QrackCircuit:
+    """Convert a Qiskit QuantumCircuit to a QrackCircuit
+
+    Args:
+        circ: QuantumCircuit to translate to QrackCircuit
+    """
+    return QrackCircuit.in_from_qiskit_circuit(circ)
+
+
+def qrack_to_qiskit(circ: QrackCircuit) -> QuantumCircuit:
+    """Convert a QrackCircuit to a Qiskit QuantumCircuit
+
+    Args:
+        circ: QrackCircuit to translate to QuantumCircuit
+    """
+    circ.out_to_file("_temp_qrack_circuit.mux")
+    return QrackCircuit.file_to_qiskit_circuit("_temp_qrack_circuit.mux")
```

Comment:
Does this file stick around after the conversion, in which case, should we either have a mechanism for removing this file or for adding it to the `.gitignore`?

Reply:
I changed this to remove the file after loading it into a Qiskit circuit. (`.gitignore` could not expect to find the temporary file in any one specific directory.)